docs/reference/layer-specification.md

```diff
@@ -254,8 +254,11 @@ log-targets:
     # - loki: Use the Grafana Loki protocol. A "pebble_service" label is
     #   added automatically, with the name of the Pebble service as its value.
     # - opentelemetry: Use the OpenTelemetry protocol (OTLP). A "service.name"
-    # label is added automatically, with the name of the Pebble service as its
-    # value.
+    #   label is added automatically, with the name of the Pebble service as its
+    #   value.
+    # - syslog: Use the syslog protocol. A "${SDID}@28978" is added as SD-ID.
+    #   28978 is Canonical's enterprise number. ${SDID} has default value
+    #   "pebble" and can be override by custom name.
```

Comment:
Thanks for updating this. However, let's just mention `pebble` explicitly here, as the *user* can't change that. And also let's mention the service name comes through as the syslog APP-NAME field. Something like:
```suggestion
    # - syslog: Use the syslog protocol. The syslog SD-ID field is set to "pebble@28978"
    #   (28978 is Canonical's enterprise number). The syslog APP-NAME field is set to
    #   the Pebble service name.
```